instrumentation/instasarama/partition_consumer_test.go

```diff
@@ -54,12 +54,12 @@ func TestPartitionConsumer_Messages(t *testing.T) {
 	var collected []*sarama.ConsumerMessage
 	timeout := time.After(1 * time.Second)
 
-CONSUMER_LOOP:
+ConsumerLoop:
```

Comment:
I'd rather suggest we keep the label the way it is to not to confuse with a variable/constant.

Reply:
Fine for me. I'll revert